src/snapred/backend/recipe/algorithm/DetectorPeakPredictor.py

```diff
@@ -18,6 +18,7 @@ def PyInit(self) -> None:
         # declare properties
         self.declareProperty("InstrumentState", defaultValue="", direction=Direction.Input)
         self.declareProperty("CrystalInfo", defaultValue="", direction=Direction.Input)
+        self.declareProperty("PeakIntensityThreshold", defaultValue="0.05", direction=Direction.Input)
```

Comment:
```suggestion
        self.declareProperty("PeakIntensityThreshold", defaultValue=0.05, direction=Direction.Input)
```
will make the property a float so you don't have to convert it